File: web/web service/DAO/Usuario.php
<?php  

class Usuario {

	//Atributos
	private $idUsuario;
    private $nomUsuario;
    private $contrasenia;
    private $foto;
    private $cliente;

	//Constructores
    function __construct() {

	}

	function Usuario($nomUsuario, $contrasenia, $foto, $cliente) {
		this::$nomUsuario = $nomUsuario;
		this::$contrasenia = $contrasenia;
		this::$foto = $foto;
		this::$cliente = $cliente;
	}

	//Métodos constructores
	public static function getIdUsuario() {
		return this::$idUsuario;
	}

	public static function setIdUsuario($idUsuario) {
		this::$idUsuario = $idUsuario;
	}

	public static function getNomUsuario() {
		return this::$nomUsuario;
	}

	public static function setNomUsuario($nomUsuario) {
		this::$nomUsuario = $nomUsuario;
	}

	public static function getContrasenia() {
		return this::$contrasenia;
	}

	public static function setContrasenia($contrasenia) {
		this::$contrasenia = $contrasenia;
	}

	public static function getFoto() {
		return this::$foto;
	}

	public static function setFoto($foto) {
		this::$foto = $foto;
	}

	public static function getCliente() {
		return this::$cliente;
	}

	public static function setCliente($cliente) {
		this::$cliente = $cliente;
	}

}

?>

File: app/DDD-Market.docx/app/src/main/java/com/example/ddd_market/ddd_market/modelo/DAO/Usuario.java
package com.example.ddd_market.ddd_market.modelo.DAO;

/**
 * Created by bvsr9 on 26/03/2017.
 */

public class Usuario {

    private int idUsuario;
    private String nomUsuario;
    private String contraseña;
    //Huella
    private FotoCliente foto;
    private Cliente cliente;

    public Usuario() {
    }

    public Usuario(String nomUsuario, String contraseña, FotoCliente foto, Cliente cliente) {
        this.nomUsuario = nomUsuario;
        this.contraseña = contraseña;
        this.foto = foto;
        this.cliente = cliente;
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }

    public String getNomUsuario() {
        return nomUsuario;
    }

    public void setNomUsuario(String nomUsuario) {
        this.nomUsuario = nomUsuario;
    }

    public String getContraseña() {
        return contraseña;
    }

    public void setContraseña(String contraseña) {
        this.contraseña = contraseña;
    }

    public FotoCliente getFoto() {
        return foto;
    }

    public void setFoto(FotoCliente foto) {
        this.foto = foto;
    }

    public Cliente getCliente() {
        return cliente;
    }

    public void setCliente(Cliente cliente) {
        this.cliente = cliente;
    }
}
